GiveMeBadgeMAC_LINUX.sh

```diff
@@ -0,0 +1,5 @@
+set +v
+echo echo Installing requirements for givemebadge...
```

Comment:
`echo echo` misspell. It should be just one `echo`

Reply:
Fixed.